### DISCUSSION TITLE
Native Mobile Upload Functionality

### DISCUSSION THREAD

**Discussion by shivsarthak** [immich-app/immich#9418] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We do get a photo upload option on web clients but the same isn't available on mobile clients. 
This is a very common use case and should be a must have for a media management solution. Currently on mobile devices it only syncs images from the selected backup folders. 
There could also be an option to upload it on immich through the mobile device's native share functionality. This would however be more of a good to have .
In general an upload functionality should be available on mobile devices too.  

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075, #2659
